components/jenkins/role.tf

```diff
@@ -0,0 +1,30 @@
+resource "azurerm_user_assigned_identity" "usermi" {
+  resource_group_name = var.mi_rg
+  location            = var.location
+  name                = "jenkins-${var.key_vault_name}-mi"
+  tags                = module.tags.common_tags
+}
+
+resource "azurerm_role_assignment" "miroles" {
+  scope                = data.azurerm_resource_group.acr_rg.id
+  role_definition_name = "Contributor"
+  principal_id         = data.azurerm_resource_group.mi.id
```

Comment:
This should reference `DTS Platform Operations` AD group as the principal (you'll need to bring it in as a data source -- same goes for subidcontributer role assignment

Reply:
https://registry.terraform.io/providers/hashicorp/azuread/latest/docs/data-sources/groups